manifests/0000_31_cluster-baremetal-operator_05_kube-rbac-proxy-config.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
```

Comment:
i think add this to config/promethus
the point of having the config/profiles/<> is that if we made one called
config/profiles/localtest for running on your laptop and we disabled prometus you could then do as below and it would work
```
kustomize build config/profiles/localtest | kubectl -f
```

Reply:
Ah! Thanks for that pointer. I made this change.

Reply:
Done.